Tools/wasm/emscripten/__main__.py

```diff
@@ -215,12 +215,29 @@ def configure_emscripten_python(context, working_dir):
     exec_script = working_dir / "python.sh"
     exec_script.write_text(
         dedent(
-            f"""\
+            """\
```

Comment:
Are you sure this shouldn't be an f-string?
```suggestion
            f"""\
```

Reply:
Right...